### PR TITLE
fix(Text/native): adds text-align prop

### PR DESCRIPTION
**Pull request by shridhar52war**

### CODE REVIEW

File: src/atoms/Text/Text.native.js [hunk 1]
```diff
@@ -32,6 +32,12 @@ const styles = {
       return getLineHeight(theme, size);
     }
   },
+  textAlign({ align }) {
+    if (align == 'inherit' || align == 'initial') {
```
Comment (deorukhkarchaitanya) [inline src/atoms/Text/Text.native.js:36]: Shouldn't we use ```===```?
```suggestion
    if (align === 'inherit' || align === 'initial') {
```

Comment (dhruvdutt) [reply src/atoms/Text/Text.native.js:36]: Don't we have this check as eslint rule? 🤦‍♂

File: src/atoms/Text/Text.native.js [hunk 2]
```diff
@@ -32,6 +32,12 @@ const styles = {
...
+      return 'auto';
```
Comment (dhruvdutt) [inline src/atoms/Text/Text.native.js:37]: Is this because React Native doesn't have `inherit` or `initial`?

Comment (shridhar52war) [reply src/atoms/Text/Text.native.js:37]: RN supports `'auto', 'left', 'right', 'center', 'justify' `

File: src/atoms/Text/Text.native.js (hunk 1 above)
Comment (kamleshchandnani) [reply src/atoms/Text/Text.native.js]: We do. Something wrong with Shridhar's machine. I guess I need to now run linters in CI as well

**Pull request merged**